src/main/java/lotto/automatic/domain/LottoRank.java

```diff
@@ -0,0 +1,30 @@
+package lotto.automatic.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+
+    FIRST(6, 200000000L),
```

Comment:
Java 7 버전 부터 숫자형에 대해 언더스코어를 활용하여 가독성을 높일 수 있게 되었습니다. 
```suggestion
    FIRST(6, 200_000_000L), 
```
와 같이 표현하여 가독성을 높여보세요 !

Reply:
언더스코어 추가하였습니다.